controllers/v1/observationsController.js

```diff
@@ -678,6 +685,26 @@ module.exports = class Observations extends Abstract {
                         message: responseMessage 
                     });
                 }
+                
+                let programQueryObject = {
```

Comment:
@aman-tunerlabs - Do we need this? Don't we get all program information from the solution itself ?

Reply:
@akash-above  I made a query to program documents based on assessment/details api where we are projecting imageCompression . If imageCompression is not required yes we can get information from solution.